test/sql-luatest/metadata_test.lua

```diff
@@ -0,0 +1,305 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group('gh-4755', {{engine = 'memtx'}, {engine = 'vinyl'}})
+
+g.before_all(function(cg)
+    cg.server = server:new({alias = 'master'})
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:stop()
+end)
+
+local g2 = t.group("full_metadata", {{remote = true}, {remote = false}})
```

Comment:
Let's use
```Lua
local g = t.group("full_metadata", {{remote = true}, {remote = false}})
```

After this test we'll define the group for the second test as
```Lua
g = t.group('gh-4755', {{engine = 'memtx'}, {engine = 'vinyl'}})
```

That is, I mean moving the definition of the second group immediately before the test that uses it.